epitator/importers/import_geonames.py

```diff
@@ -38,7 +38,22 @@
 ]
 
 
-def read_geonames_csv():
+def windows_max_int():
+    # Find largest integer that does not cause an OverflowError for csv.field_size_limit using "binary search"
+    top = 2**32  # Largest int in Python has 64 bit but largest int in Windows has 32 bit
+    lower = 2**30  # The largest int is between 2^30 and 2^32
+    mid = (top + lower) // 2
+    while top - lower != 1:
+        try:
+            mid = (top + lower) // 2
+            csv.field_size_limit(mid + 1)  # Find the mid that barely causes an OverflowError
```

Comment:
I think there might be an edge case remaining here. If mid was less than top by one here, the return value would be 1 less than the largest field size limit determined to be valid.

Reply:
Hello Nathan, thanks for your reply. I embarrassedly need to admit that I don't know how to fix it in a pretty way. I could implemnt something like this in the end to find the absolut maximum number valid:
```python
for i in range(top, (lower-1), -1):
    try:
        csv.field_size_limit(i)
        break
    except OverflowError:
        continue
```
But since the largest usable bit size for my machine is for example 2147483646, don't you think that one bit less would not be a meaningful difference and we should rather keep the code simple?

Reply:
Or I could make it easier and use 
```python 
max_c_long_on_windows = (2**32 / 2) - 1
```
and just assume that this will always hold. 
Since I have no clue about operating systems I don't know whether this is a valid option.
I used this the formular from https://medium.com/@nickteixeira/how-to-explain-to-my-wife-what-i-do-how-do-you-get-the-maximum-and-minimum-values-for-integer-befdc263a3a2

Reply:
@aauss Are you able to replace the entire binary search with the max_c_long_on_windows approach? That seems like a win to me. You're checking the platform is win32 and I would be surprised if it doesn't hold when the platform stays the same. Just curious, are you using python 2 or 3?

Reply:
I am using Python 3 and no problem, I will replace the code then.